logging/jvm/src/test/scala/com/ccadllc/cedi/dtrace/logging/JsonLogEncodingTest.scala

```diff
@@ -33,7 +33,7 @@ class JsonLogEncodingTests extends WordSpec with TestSupport {
 
   // format: OFF
   val calculateQuarterlySalesTraceContextJson = Json.obj(
-    "where"           -> calculateQuarterlySalesTraceContext.system.metadata.asJson,
+    "where"           -> calculateQuarterlySalesTraceContext.system.data.combinedStrMap.asJson,
```

Comment:
Should these keys use _ to be ECS compliant?

Reply:
these wouldn't necessarily (since the logging module isn't ECS compliant) but the EcsLogstashLogbackEmitter should be.